docs/main.css

```diff
@@ -0,0 +1,376 @@
+@import url("../utils/reset.css");
+@import url("../utils/color.css");
+@import url("../utils/utility.css");
+
+/* genral styles for documentation page */
+:root {
+  --font-poppins: "Poppins", sans-serif;
+  --font-montserrat: "Montserrat", sans-serif;
+  --cta-bgcolor: var(--DARK-BASE);
+  --cta-fcolor: var(--NEUTRAL);
+  --heading--color: var(--DARK-BASE);
+  --author-name-color: var(--DARK-BASE);
+  --azure: azure;
+}
+
+/*setting default font-size to 16px*/
+html {
+  font-size: 62.5%;
+  font-family: var(--font-montserrat);
+}
+
+body {
+  font-size: 1.6rem;
+}
+
+/* Layout */
+.container {
+  max-width: 144rem;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+}
+
+/* header */
+.header {
+  font-size: 1.6rem;
+  background: var(--NEUTRAL);
+  opacity: 0.9;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  justify-content: space-between;
+  font-family: var(--font-poppins);
+  box-shadow: 0px 0px 9px 3px rgba(41, 41, 41, 0.25);
+}
+
+
+
+.hamburger, .close{
+  display: none;
+}
+
+.header a{
+  text-decoration: none;
+}
+
+.logo {
+  color: var(--DARK-BASE);
+  font-weight: 600;
+  padding: 0 1rem;
+  font-size: 1.8rem;
+  padding: 1rem;
+  margin-inline-start: 1rem;
+}
+
+/* ul === .navitems__container */
+.header .navitems__container {
+  margin-inline-start: auto;
+  padding: 0 1rem;
+  gap: 4rem;
+}
+
+.navitems__container a {
+  text-decoration: none;
+  color: black;
+  font-weight: 600;
+}
+
+
+.navitems__container a::after {
+  content: "";
+  width: 0%;
+  display: block;
+  margin: auto;
+  height: 2px;
+  background: var(--LIGHT-ACCENT);
+}
+
+.navitems__container a:hover::after {
+  display: block;
+  width: 100%;
+  transition: 0.5s;
+}
+
+
+
+.navitems__container li:nth-of-type(3) span:hover {
+  background: var(--LIGHT-GRAY);
+}
+
+.navitems__container li:nth-of-type(3) a:hover::after {
+  width: 0;
+}
+
+/* main */
+main {
+  background: url("./assets/cover.jpg");
+  background-size: cover;
+  padding: 2rem;
+  min-block-size: 100vh;
+  line-height: 1.1;
+  background: azure;
+}
+
+.container > * {
+  margin-block: 0;
+}
+
+.container > main {
+  margin-block: auto;
+}
+
+.container > :first-child:not(main) {
+  margin-block-start: 0;
+}
+
+.container > :last-child:not(main) {
+  margin-block-end: 0;
+}
+
+/* cover content */
+.main__content {
+  margin-inline-start: 0.8rem;
+  margin-block-start: 8rem;
+}
+
+main .title {
+  font-size: 12rem;
+  word-wrap: break-word;
+  color: var(--heading--color);
+  font-weight: 600;
+}
+
+main .slogan {
+  font-size: 3.2rem;
+  word-wrap: break-word;
+}
+
+.info {
+  margin-block-start: 2rem;
+}
+
+main .about {
+  font-size: 2rem;
+  max-width: 70rem;
+  word-wrap: break-word;
+}
+
+main .author-name {
+  margin-block-start: 2rem;
+  color: var(--author-name-color);
+  font-size: 2.4rem;
+}
+
+.author-name a:active{
+  color: var(--DARK-BASE);
+}
+
+
+main {
+  display: flex;
+  flex-direction: column;
+  gap: 4rem;
+}
+
+.cta_buttons {
+  margin-inline-start: 0.4rem;
```

Comment:
can use px here

Reply:
Thanks for pointing this, I would definitely be going to miss that :->